### ISSUE TITLE
Crashing at starting

### ISSUE THREAD

**Issue by TheButterbrotMan** [Spaceman/FabricSecretRooms#14]
https://bytebin.lucko.me/HdB6caCiHi

**Comment by TheButterbrotMan** [comment]
https://github.com/Bstn1802/AutoReconnect/issues/19

**Comment by Bstn1802** [comment]
Quick statement, I have nothing to do with this error in Servercrash log, only with the one in Clientcrashlog (mentioned in the other issue mentioned here) but you don't have anything to do with that either, they're completely unrelated.
Small hint, this issue here has something to do with your LanternButtonBlock, which minecraft apparently seems to be confusing with an air block and therefore can't set the direction property on it at which point I'm asking myself if the lanternblock even has a direction property, like why would it need one? Anyways, you have to take it from there, more Info in the crash log, at least I hope you can get more out of it that I could.

**Comment by Spaceman** [comment]
Yeah I have a feeling it's just deprecated methods I was too lazy to fix

**Comment by TheRanker** [comment]
this crash showing Dim Doors.. is actually caused by Oh_The_Biomes_You'll_Go-fabric-1.18.2-1.4-RC-5.jar
using this version of byg and DimensionalDoors-4.0.1-1.18.2.jar  .. gives an the servercrash log indicated in first post.

**Comment by ThePoultryMan** [comment]
> edit: Lanterns Belong on Walls by CoalTheBurnRock also causes this same crash about dim doors.

This mod overrides part of the mixin that I do to lanterns, and this was what was causing it to crash (with Lanterns Belong on Walls). This will be fixed in the next update.

**Comment by Spaceman** [comment]
So this is resolved?

**Comment by ThePoultryMan** [comment]
It should be. The crash log definitely points to my mod causing the crash, and I released an update to fix the crash. So, I think we're all good.
